### PR TITLE
exclude --module-rules-options from schema for cli

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
Cli related change

**Did you add tests for your changes?**
Yes

**Does this PR introduce a breaking change?**
No

**What needs to be documented once your changes are merged?**
Probably nothing

### CODE REVIEW

File: schemas/WebpackOptions.json [hunk 1]
```diff
@@ -2487,6 +2487,9 @@
       "properties": {
         "compiler": {
           "description": "Match the child compiler name.",
+          "cli": {
+            "exclude": true
```
Comment (sokra) [inline schemas/WebpackOptions.json:2491]: Whats the motivation of removing all rule related options from the CLI?

The old cli args also allow to specify that e. g. via `--module-bind`.

Is it only of aesthetics? Maybe we can flag them with `"level": "advanced"` instead and omit them in `--help` and only show them in `--help=advanced`

Comment (snitin315) [reply schemas/WebpackOptions.json:2491]: > Maybe we can flag them with "level": "advanced" instead and omit them in --help and only show them in --help=advanced

I personally like this idea, I think for now we can support all the flags and we can implement this based on feedback and discussion later.

Comment (evilebottnawi) [reply schemas/WebpackOptions.json:2491]: Agree, about `--help=advanced`.

How do you see working with these flags?

**Comment by sokra** [comment]
Thanks

**Pull request merged**